docs/configuration.md

```diff
@@ -64,6 +64,8 @@ Iceberg tables support table properties to configure table behavior, like the de
 | write.orc.block-size-bytes         | 268435456 (256 MB) | Define the default file system block size for ORC files |
 | write.orc.compression-codec        | zlib               | ORC compression codec: zstd, lz4, lzo, zlib, snappy, none |
 | write.orc.compression-strategy     | speed              | ORC compression strategy: speed, compression |
+| write.orc.bloom.filter.columns     | (not set)          | Comma separated list of column names for which a Bloom filter must be created |
+| write.orc.bloom.filter.fpp         | 0.05               | False positive probability for Bloom filter (must > 0.0 and < 1.0) |
```

Comment:
You might want to match the parquet configurations a bit more closely.

They are
```
| write.parquet.bloom-filter-enabled.column.col1          | (not set) | Enables writing a bloom filter for the column: col1|
| write.parquet.bloom-filter-max-bytes | 1048576 (1 MB)   | The maximum number of bytes for a bloom filter bitset |
```

So you could do `write.orc.bloom-filter-enabled.column.col1`. This also matches other config value formatting that's per column, such as `write.metadata.metrics.column.col1`.

For the `fpp`, as that seems to be how the value is set on the ORC bloom filter, I would suggest keeping it that way. But if the parquet implementation is translating from the max-bytes to fpp, then possibly setting the config that way for consistency (but I doubt that it is).

Reply:
IMO, in this PR, we align `write.parquet.bloom-filter-enabled.column` and `write.orc.bloom.filter.columns` with native parquet and orc options. 

We can create a new issue to unify these options. 
In the new issue, we discuss a new option of no parquet and orc tag like `write.rowgroup.bloom-filter.columns` or `write.rowgroup.bloom-filter-enabled.column` to unify file format options. 
Maybe ·write.parquet.bloom-filter-enabled.column = id, name· or `[id, name]` is more elegant than `write.rowgroup.bloom-filter.columns.id = true`.

PS: 
I find the Trino iceberg connector use the style like `orc_bloom_filter_columns = [id, name]`
https://trino.io/docs/current/connector/iceberg.html#iceberg-table-properties

I'm happy to diver this.
@kbendick What do you think?

Reply:
I’m good with following up on anything in a new issue.

This is almost there and we should leave it as is (minus any nits). Using configs similar to those offered by native file formats is a good call imo.